django_ibmi/schema.py

```diff
@@ -46,6 +41,18 @@ class DB2SchemaEditor(BaseDatabaseSchemaEditor):
     sql_drop_pk = "ALTER TABLE %(table)s DROP PRIMARY KEY"
     sql_drop_default = "ALTER TABLE %(table)s ALTER COLUMN %(column)s DROP DEFAULT"
 
+    def quote_value(self, value):
+        if isinstance(value, (datetime.date, datetime.time, datetime.datetime)):
+            return "'%s'" % value
+        elif isinstance(value, str):
+            return "'%s'" % value.replace("\'", "\'\'").replace('%', '%%')
```

Comment:
```suggestion
            return "'%s'" % value.replace("'", "''").replace('%', '%%')
```

single quotes don't need escaping in double quoted strings.

Reply:
Also, should we be escaping percent signs? This would only matter if it's being passed to a `LIKE` clause. Otherwise, it would mess up the query, I'd think.